data/mods/Aftershock/martialarts.json

```diff
@@ -0,0 +1,79 @@
+[
+  {
+    "id": "style_aikido",
+    "copy-from": "style_aikido",
+    "type": "martial_art",
+    "name": "Aikido",
+    "extend": { "weapons": [ "afs_hydraulic_gauntlet" ] }
+  },
+  {
+    "id": "style_eskrima",
+    "copy-from": "style_eskrima",
+    "type": "martial_art",
+    "name": "Eskrima",
+    "extend": { "weapons": [ "ceramic_knife" ] }
+  },
+  {
+    "id": "style_fencing",
+    "copy-from": "style_fencing",
+    "type": "martial_art",
+    "name": "Fencing",
+    "extend": { "weapons": [ "afs_energy_saber_on" ] }
+  },
+  {
+    "id": "style_medievalpole",
+    "copy-from": "style_medievalpole",
+    "type": "martial_art",
+    "name": "Fior Di Battaglia",
+    "extend": { "weapons": [ "afs_titanium_bat" ] }
+  },
+  {
+    "id": "style_judo",
+    "copy-from": "style_judo",
+    "type": "martial_art",
+    "name": "Judo",
+    "extend": { "weapons": [ "afs_hydraulic_gauntlet" ] }
```

Comment:
See Aikido comment.

Reply:
I try and help you and now all my tests are failing ;P.  But yeah I just wandered through and tossed in whatever might possibly fit.  I've removed your requests.